src/Screens/Onboarding/SignIn.tsx

```diff
@@ -39,8 +41,8 @@ const SignIn = (props: Props) => {
     props.navigation.goBack();
   };
 
-  const onLogin = (credentials: authApi.Credentials) => {
-    props.login(credentials);
+  const onLogin = (creds: authApi.Credentials) => {
+    props.login(creds);
```

Comment:
This changed because I had some extra logic on this PR when I had admin-login-functionality here. I'll just rename it back